README.md

```diff
@@ -80,4 +94,4 @@ Learn how to setup Ruby logging by exploring the workings of the [example projec
  
 ## Get in touch
 
-Have any questions? Please explore the Logtail [documentation](https://docs.logtail.com/) or contact our [support](https://betterstack.com/help).
+Have any questions? Please explore the Better Stack Logs [documentation](https://betterstack.com/docs/logs/) or contact our [support](https://betterstack.com/help).
```

Comment:
```suggestion
Have any questions? Please explore the Better Stack [documentation](https://betterstack.com/docs/logs/) or contact our [support](https://betterstack.com/help).
```

Reply:
We are trying to use just `Better Stack` except in cases where it would be very confusing. We want people to think about Better Stack as one product/brand.